source/apiVolontaria/apiVolontaria/urls.py

```diff
@@ -77,6 +78,11 @@
         r'^location/',
         include('location.urls', namespace="location"),
     ),
+    url(
+        r'^info-page$',
+        InfoPageView.as_view(),
+        name='info_page',
+    ),
```

Comment:
Please do a redirection to a new `urls.py` file inside the new app to keep trace of url standard and easy to find.

You can keep example on `location`

Reply:
Good point ! I've also modify the route to have pages/info so if in the futur there is other king of pages, it will be ready to accept it.